src/content/app/tools/blast/components/blast-submission-header-container/BlastSubmissionHeaderGrid.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { type ReactNode } from 'react';
+
+import styles from './BlastSubmissionHeaderGrid.scss';
+
+/**
+ * This is a dumb component whose sole purpose is to arrange the children into a grid
+ */
+
+// TODO: consider which interface is better: a single "children" prop,
+// or individual "first", "second", "third" props. With individual props,
+// the consumer of the component will know that this component accepts only three children
+type Props = {
+  children: ReactNode;
```

Comment:
I think this is fine as long as there is only one consumer.

Reply:
There will be at least two: a ListedBlastSubmission, and the detailed BLAST submission: https://xd.adobe.com/view/f523ac5e-275c-4835-9d91-33910bed4ef3-a070/screen/05953e96-64db-41c5-9b26-96febff44a24

Reply:
I was wondering it it can be changed to `name`, `details` and `action` sections but it won't make sense to have `edit/rerun` within the details section. 

I'm OK with leaving it as it is in this PR and perhaps come back to it when we work on the detailed submission view.